eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -473,10 +471,8 @@ public function updateVersion($contentId, $versionNumber, Request $request)
 
         try {
             $this->repository->getContentService()->updateContent($versionInfo, $contentUpdateStruct);
-        } catch (ContentValidationException $e) {
-            throw new BadRequestException($e->getMessage());
         } catch (ContentFieldValidationException $e) {
-            throw new BadRequestException($e->getMessage());
+            throw new RESTContentFieldValidationException($e->getFieldErrors(), $e->getMessage());
```

Comment:
While on it make it take the whole exception, like other exceptions do. cause it's the code, trace, file and line of the original exception we care about, not the one here.